perf_test/sparse/KokkosSparse_spmv_struct_tuning.cpp

```diff
@@ -605,7 +605,7 @@ int main(int argc, char** argv) {
       const double alpha = 1.0, beta = 1.0;
       size_t bufferSize     = 0;
       void* dBuffer         = NULL;
-      cusparseSpMVAlg_t alg = CUSPARSE_MV_ALG_DEFAULT;
+      cusparseSpMVAlg_t alg = CUSPARSE_SPMV_ALG_DEFAULT;
```

Comment:
My guess is this will fail with older version of cusparse, you probably need this instead:

```
#if CUSPARSE_VERSION < 11000
cusparseS[MVAlg_t alg = CUSPARSE_MV_ALG_DEFAULT;
#else if CUSPARSE_VERSION >= 11000
cusparseSpMVAlg_t alg = CUSPARSE_SPMV_ALG_DEFAULT;
#endif

Reply:
So I looked through the documentation, and it seems like the first place this was deprecated was in CUDA 11.2.1. 
(See here for 11.2.0: https://docs.nvidia.com/cuda/archive/11.2.0/cusparse/index.html and here for 11.2.1: https://docs.nvidia.com/cuda/archive/11.2.1/cusparse/index.html Search for CUSPARSE_MV_ALG_DEFAULT.)

So does that mean to use `#if CUSPARSE_VERSION < 11201`  Or did I not translate that value correctly?

Reply:
Note: from `cusparse.h`
```  58 //##############################################################################
  59 //# CUSPARSE VERSION INFORMATION
  60 //##############################################################################
  61 
  62 #define CUSPARSE_VER_MAJOR 11
  63 #define CUSPARSE_VER_MINOR 4
  64 #define CUSPARSE_VER_PATCH 1
  65 #define CUSPARSE_VER_BUILD 1152
  66 #define CUSPARSE_VERSION (CUSPARSE_VER_MAJOR * 1000 + \
  67                           CUSPARSE_VER_MINOR *  100 + \
  68                           CUSPARSE_VER_PATCH)

Reply:
Yes, so that shows the formula and the macro name which is good to verify. If the changes where introduce originally in cuda/11.2.0 then `#if CUSPARSE_VERSION => 11200` is the correct way to branch between old and new verisons.